.circleci/config.yml

```diff
@@ -0,0 +1,80 @@
+# This file was generated by the excavator check 'excavator/manage-circleci' as specified in .circleci/template.sh.
+# To request a modification to the general template, file an issue on Excavator.
+# To manually manage the CircleCI configuration for this project, remove the .circleci/template.sh file.
+
+version: 2
+jobs:
+  build:
+    docker:
+      - image: ellisjoe/openjdk-java-8-openssl:0.0.2
+    resource_class: xlarge
+    environment:
+      GRADLE_OPTS: -Dorg.gradle.console=plain -Dorg.gradle.internal.launcher.welcomeMessageEnabled=false
+
+    steps:
+      - checkout
+      - restore_cache: { key: 'gradle-wrapper-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}' }
+      - restore_cache: { key: 'gradle-cache-{{ checksum "versions.props" }}-{{ checksum "build.gradle" }}' }
+      - run: ./gradlew --parallel --stacktrace resolveConfigurations
+      - save_cache:
+          key: gradle-wrapper-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}
+          paths: [ ~/.gradle/wrapper ]
+      - save_cache:
+          key: gradle-cache-{{ checksum "versions.props" }}-{{ checksum "build.gradle" }}
+          paths: [ ~/.gradle/caches ]
+      - run: ./gradlew --parallel --continue build
+      - run:
+          command: mkdir -p ~/junit && find . -type f -regex ".*/build/.*TEST.*xml" -exec cp --parents {} ~/junit/ \;
+          when: always
+      - store_test_results: { path: ~/junit }
+
+      - deploy:
+          command: |
+            # publishing snapshots to bintray does not work, so we only publish from tag builds (not develop)
+            if [[ "${CIRCLE_TAG}" =~ [0-9]+(\.[0-9]+)+(-[a-zA-Z]+[0-9]*)* ]]; then
+              ./gradlew --stacktrace --continue publish
```

Comment:
gotta make sure `publish` will actually run `bintrayUpload`

(one option: https://github.com/palantir/conjure-java-runtime/blob/develop/gradle/publish-jar.gradle)

would need rootProject.name = 'hadoop-crypto' in your settings.gradle